pkg/cmd/urlcheck/BUILD.bazel

```diff
@@ -5,7 +5,7 @@ go_library(
     srcs = ["main.go"],
     importpath = "github.com/cockroachdb/cockroach/pkg/cmd/urlcheck",
     visibility = ["//visibility:private"],
-    deps = ["//pkg/cmd/urlcheck/lib/urlcheck"],
+    deps = ["@com_github_cockroachdb_cockroach//pkg/cmd/urlcheck/lib/urlcheck:go_default_library"],
```

Comment:
What's going on here? If Gazelle inserted this, it seems wrong.

```
bazel query @com_github_cockroachdb_cockroach//...
ERROR: No such repository '@com_github_cockroachdb_cockroach'
ERROR: no targets found beneath ''
Loading: 0 packages loaded
```

Reply:
i.e., the original `dep` was good, this definitely shouldn't have been changed

Reply:
That was gazelle. Let me poke it more to see the exact issue. Without this change the lint test was failing.

Reply:
You can probably do a `# gazelle:resolve go github.com/cockroachdb/cockroach/pkg/cmd/urlcheck/lib/urlcheck //pkg/cmd/urlcheck/lib/urlcheck` in `BUILD.bazel` (or similar, don't remember the exact syntax) to quash this.

Reply:
Thank you for the tip. It just worked fine. \o/